code/modules/research/techweb/all_nodes.dm

```diff
@@ -2231,9 +2231,11 @@
 		"donksoft_refill",
 		"donksofttoyvendor",
 		"largecrossbow",
+		/* BUBBERSTATION CHANGE START: REMOVES AUTORIFLE DESIGNS.
 		"mag_autorifle",
 		"mag_autorifle_ap",
 		"mag_autorifle_ic",
+		BUBBERSTATION CHANGE END. */
```

Comment:
Modularize this inside ``modular_zubbers\code\modules\research\techweb\all_nodes.dm``

Reply:
I personally don't think that's good practice because if they ever change those ids, there would be issues.

Reply:
It'll realistically be on my end to deal with. There's no 100 percent way to stop all conflicts. Throw it inside a modular proc, even if it's inside ``modular_zubbers/modules/ballstics/wt_autorifle.dm``. Historically on skyrat not modularizing techweb edits has caused way more conflicts than the occasional two-second edit if TG decides to ever change these.

Reply:
fixed